modules/fedimint-ln-client/src/pay.rs

```diff
@@ -48,11 +47,19 @@ const RETRY_DELAY: Duration = Duration::from_secs(1);
 #[derive(Debug, Clone, Eq, PartialEq, Hash, Decodable, Encodable)]
 pub enum LightningPayStates {
     CreatedOutgoingLnContract(LightningPayCreatedOutgoingLnContract),
-    Canceled,
+    FundingRejected,
     Funded(LightningPayFunded),
     Success(String),
+    #[deprecated(
+        since = "0.4.0",
+        note = "Pay State Machine skips over this state and will retry payments until cancellation or timeout"
+    )]
```

Comment:
Yeah its to save migration code. `Refunded` can be deleted, but then we'd need to write a migration for the `Failure` state. `Refundable` can also be deleted, but then we'd need to add another migration, and since it has the same logic as `Funded` now, I think its easier just to let the state machine execute and take care of the transition.